File: src/chapters/chapter-04/csharp/Models/Product.cs
using System.ComponentModel;

namespace AIShoppingAssistant.Models;

public class Product
{

    [Description("name")]
    public string Name { get; set; }

    [Description("Product code.")]
    public string ProductCode { get; set; }

    [Description("Product description.")]
    public string Description { get; set; }

    [Description("Product price.")]
    public decimal Price { get; set; }

    [Description("Product category.")]
    public string Category { get; set; }

    [Description("Product brand.")]
    public string Brand { get; set; }
}

File: src/chapters/chapter-04/python/models/product.py
from dataclasses import dataclass, field

@dataclass
class Product:
    name: str = field(metadata={"description": "name"})
    product_code: str = field(metadata={"description": "Product code."})
    description: str = field(metadata={"description": "Product description."})
    price: float = field(metadata={"description": "Product price."})
    category: str = field(metadata={"description": "Product category."})
    brand: str = field(metadata={"description": "Product brand."})